script/cibuild

```diff
@@ -2,16 +2,16 @@
 
 set -e
 
-. script/bootstrap
+. script/common
 
 echo "Verifying code is formatted via 'gofmt -s -w  go/'"
-gofmt -s -w  go/
-git diff --exit-code --quiet
+$DGO gofmt -s -w go
+git diff --exit-code --quiet go
```

Comment:
This bit me a few times. Sometimes I have non-go changes locally and this currently checks all stuff. Not just any go code changes.